recipes/_config.rb

```diff
@@ -19,4 +19,12 @@
   node.override['hops']['dist_url']    = "#{node['install']['enterprise']['download_url']}/hopshadoop/hops-#{node['hops']['version']}.tgz"
   node.override['hops']['schema_dir']  = "#{node['install']['enterprise']['download_url']}/hopshadoop/hops-schemas"
   node.override['dal']['download_url'] = "#{node['install']['enterprise']['download_url']}/hopshadoop/ndb-dal-#{node['hops']['version']}-#{node['ndb']['version']}.jar"
+  # Set Hops EE version
+  version = node['hops']['version']
+  version_arr = version.split("-")
+  version = versionArr[0] + "-EE"
+  if version_arr.size > 1
+    version = version + "-" + version_arr[1]
+  end
+  node.override['hops']['version'] = version
```

Comment:
this won't have any effect on the `node['hops']['dist_url']` defined above. Same for the `node['dal']['download_url']`

Reply:
thanks, fixed